.github/workflows/code-quality.yml

```
@@ -152,38 +238,123 @@ jobs:
+              echo "⚠️ Moderate/low vulnerabilities found (non-blocking)"
+              echo "high_vulns=false" >> $GITHUB_OUTPUT
+              # Don't fail for moderate/low
+              exit 0
+            fi
+          else
+            echo "✅ No security vulnerabilities found"
+            echo "high_vulns=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Create security issue for moderate vulnerabilities
+        if: failure() == false && contains(steps.audit.outputs.*, 'moderate')
```

actionlint (1.7.7)

elements of object at receiver of object filtering `.*` must be type of object but got "string". the type of receiver was "{string => string}"

(expression)